### ISSUE TITLE
Unable to change the Accessibility Description for App menu Overlay Icon for toolbar in Android

### ISSUE THREAD

**Issue by Alekhya03** [material-components/material-components-android#3116]
Hi Team,
      I am trying to set a new description to the overlay Icon(3 dots button in the toolbar) but it's not replicating. can you help me with this?

**Comment by drchen** [comment]
Unfortunately I don't think there's a reliable way to do that. 

MaterialToolbar is just a thin overlay of AppCompat's Toolbar, which in turn uses the framework's ActionMenuView to display menu.

ActionMenuView is a LinearLayout, so you can probably try to get the last child of it, which should be the overflow button, and set the content description of it.

Hopefully this is helpful.

**Comment by Alekhya03** [comment]
As I can check we can able to get overflowIcon which can't be set the description to it

**Comment by drchen** [comment]
You can try to iterate over the children of Toolbar, find an instance of ActionMenuView, and get the last child view of it. 
This view is supposed to be the overflow icon view and you can set content description onto it. 

That's the best way I can think of. If that doesn't work I'm afraid there's no better way to do that. It's a framework restriction unfortunately.
